taxi_service/settings.py

```diff
@@ -39,11 +39,13 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
+    "debug_toolbar",
```

Comment:
The `debug_toolbar` is included in `INSTALLED_APPS`. Ensure that this is only enabled in a development environment, as it can expose sensitive information in production.